src/replay/docs/advanced-playback/bitrates.mdx

```diff
@@ -4,6 +4,108 @@ route: /advanced-playback/bitrates
 menu: 'Advanced playback'
 ---
 
+import { Playground, PropsTable } from 'docz';
+import Replay from '../../default-player/Replay';
+import HlsjsVideoStreamer from '../../components/player/VideoStreamer/HlsjsVideoStreamer/HlsjsVideoStreamer';
+import ShakaVideoStreamer from '../../components/player/VideoStreamer/ShakaVideoStreamer/ShakaVideoStreamer';
+import CompoundVideoStreamer from '../../components/player/VideoStreamer/CompoundVideoStreamer/CompoundVideoStreamer';
+import '../../replay-default.css';
+
 # Bitrates and adaptive quality selection
 
-This chapter is not written yet, but will discuss how an underlying adaptive streaming library's quality management is exposed in Replay.
+Bitrates and adaptive quality selection in Replay is supported through third party player integrations, namely several different `<...VideoStreamer/>` components included in the Replay package. Two of these integrate the most common and mature open source alternatives,
+HLS.js and Shaka Player. Both adaptive quality selection and manual quality selection options are available in Replay. Also, there are some options to control bitrates selection both in adaptive quality selection and manual quality selection.
+
+Examples of the different bitrates and adaptive quality selection use case might be for both HLS and DASH streams.
+
+## Adaptive quality selection
+
+### Automatic selection
+
+By default, Replay handles quality switch automatically, using heuristics based on fragment loading bitrate and quality level bandwidth exposed in the variant manifest.
+
+The following example contains `QualitySelection` options in Replay and hovering over the options there will be available bitrates of the stream and an indicator will point
+to the currently streaming bitrate. 
+
+<Playground>
+  <Replay
+    source={{
+             streamUrl: 'public/example-media/adaptive.m3u8',
+             contentType: 'application/x-mpegurl'
+           }}
+    initialPlaybackProps={{ isPaused: true }}>
+    <CompoundVideoStreamer/>
+  </Replay>
+</Playground>
+
+### Capped selection
+
+From the available bitrates in `QualitySelection` options in Replay, When a specific bitrate is selected then it will indicate a maximum bitrate to be considered for adaptive
+bitrate switching.
+
+The following example contains `QualitySelection` options in Replay and hovering over the options there will be available bitrates of the stream and an indicator will point
+to the current streaming bitrate. Then a specific bitrate can be selected and Replay will consider it the maximum bitrate for adaptive switching.
+
+<Playground>
+  <Replay
+    source={{
+             streamUrl: 'public/example-media/adaptive.m3u8',
+             contentType: 'application/x-mpegurl'
+           }}
+    initialPlaybackProps={{ isPaused: true }}>
+    <CompoundVideoStreamer/>
+  </Replay>
+</Playground>
```

Comment:
Looks like this example code is identical to the first example code in the previous chapter. According to the description I believe this is an error?

Reply:
Yes, you are right. It's the same code example as for Adaptive quality selection we don't need to override any config. I put this example for different available options in Adaptive quality selection. I changed some lines to reflect it. What do you think about it?

Reply:
I would think the doc reader looks for differences in the two examples, and wouldn't find anything different in the player or the code (unless actually choosing the different options), so it could still leave them a bit confused.

In my opinion both cases (automatic & capped selection) could and should be described referring to the same player instance and code, like:

```suggestion
## Adaptive quality selection

### Automatic selection

By default, Replay handles quality switch automatically, using heuristics based on fragment loading bitrate and quality level bandwidth exposed in the variant manifest.

The following example contains `QualitySelection` options in Replay and hovering over the options there will be available bitrates of the stream and an indicator will point
to the currently streaming bitrate. 

<Playground>
  <Replay
    source={{
             streamUrl: 'public/example-media/adaptive.m3u8',
             contentType: 'application/x-mpegurl'
           }}
    initialPlaybackProps={{ isPaused: true }}>
    <CompoundVideoStreamer/>
  </Replay>
</Playground>

### Capped selection

When a specific bitrate is selected from the options mentioned above, it will indicate a maximum bitrate to be considered for adaptive bitrate switching. Replay will consider it as the maximum bitrate for adaptive switching.
```